test/integration/audit_test.rb

```diff
@@ -1,4 +1,4 @@
-require 'test_helper'
+require 'integration_test_helper'
 
 class AuditIntegrationTest < ActionDispatch::IntegrationTest
```

Comment:
This class also has a JS-enabled test, could you either move it or define a second class in this file for it?